health_check.sh

```diff
@@ -65,14 +65,11 @@ fi
 if [[ ! -z "${http_publish_uri}" ]]
 then
 	check_url="${proto}"://"${http_publish_uri}"
-else
-	echo "not possible to get Graylog listen URI - abort"
-	exit 1
 fi
 
 
 if curl --silent --fail "${check_url}"/api
 then
   	exit 0
 fi
```

Comment:
May want to check if you have a `check_url` before calling curl for better logging.
```
if [[ -z "${check_url}" ]]
then
	echo "Not possible to get Graylog listen URI - abort"
	exit 1
fi
```

Reply:
that is a good point! 

thx